src/testdrive/src/util/text.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use atty::Stream;
+use similar::{ChangeTag, TextDiff};
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+/// Trims trailing whitespace from each line of `s`.
+pub fn trim_trailing_space(s: &str) -> String {
+    let mut lines: Vec<_> = s.lines().map(|line| line.trim_end()).collect();
+    while lines.last().map_or(false, |l| l.is_empty()) {
+        lines.pop();
+    }
```

Comment:
does this remove empty lines?

Reply:
Yeah, it removes trailing lines that consist of only whitespace.